### PR TITLE
Fix the coding style issue

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
Please ignore the below false alarm:
```
/home/xiaoxiang/backup/os/nuttx/apps/examples/camera/camera_main.c:639:44: error: Operator/assignment must be followed with whitespace
/home/xiaoxiang/backup/os/nuttx/apps/interpreters/quickjs/qjsmini.c:105:23: error: Mixed case identifier found
/home/xiaoxiang/backup/os/nuttx/apps/system/vi/vi.c:540:57: error: Multiple data definitions
```
## Impact
Minor change

## Testing
Pass CI

### CODE REVIEW

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 1]
```diff
@@ -1473,20 +1455,16 @@ static void test_nvs_corrupted_sector_close(struct mtdnvs_ctx_s *ctx)
     }
 
   printf("%s: success\n", __func__);
-
   return;
 
 test_fail:
-
   if (nvs_fd >= 0)
     close(nvs_fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1462]: Let's add `{}` here too

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 2]
```diff
@@ -1473,20 +1455,16 @@ static void test_nvs_corrupted_sector_close(struct mtdnvs_ctx_s *ctx)
...
 
   if (mtd_fd >= 0)
     close(mtd_fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1465]: Let's add `{}` here too

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 3]
```diff
@@ -1638,17 +1616,13 @@ static void test_nvs_full_sector(struct mtdnvs_ctx_s *ctx)
     }
 
   printf("%s: success\n", __func__);
-
   return;
 
 test_fail:
-
   if (fd >= 0)
     close(fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1623]: Let's add `{}` here too

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 4]
```diff
@@ -1828,20 +1802,16 @@ static void test_nvs_gc_corrupt_close_ate(struct mtdnvs_ctx_s *ctx)
     }
 
   printf("%s: success\n", __func__);
-
   return;
 
 test_fail:
-
   if (nvs_fd >= 0)
     close(nvs_fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1809]: Ditto

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 5]
```diff
@@ -1828,20 +1802,16 @@ static void test_nvs_gc_corrupt_close_ate(struct mtdnvs_ctx_s *ctx)
...
 
   if (mtd_fd >= 0)
     close(mtd_fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1812]: Ditto

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 6]
```diff
@@ -1966,17 +1936,13 @@ static void test_nvs_gc_corrupt_ate(struct mtdnvs_ctx_s *ctx)
     }
 
   printf("%s: success\n", __func__);
-
   return;
 
 test_fail:
-
   if (mtd_fd >= 0)
     close(mtd_fd);
```
Comment (pkarashchenko) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:1943]: Ditto

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 3 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 4 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 6 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 2 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 5 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

File: testing/mtd_config_fs/mtd_config_fs_test_main.c (hunk 1 above)
Comment (xiaoxiang781216) [reply testing/mtd_config_fs/mtd_config_fs_test_main.c]: Done.

**Review by jerpelea — APPROVED** [review]

**Comment by pkarashchenko** [comment]
What about this error?
```
Error: /home/runner/work/incubator-nuttx-apps/incubator-nuttx-apps/apps/testing/mtd_config_fs/mtd_config_fs_test_main.c:1473:1: error: Blank line contains
```

**Comment by xiaoxiang781216** [comment]
> What about this error?
> 
> ```
> Error: /home/runner/work/incubator-nuttx-apps/incubator-nuttx-apps/apps/testing/mtd_config_fs/mtd_config_fs_test_main.c:1473:1: error: Blank line contains
> ```

Done.

**Review by jerpelea — APPROVED** [review]

**Pull request merged**